src/libs/SelectionScraper/index.js

```diff
@@ -88,6 +89,10 @@ const replaceNodes = (dom) => {
     let domChildren;
     const domAttribs = {};
 
+    if (dom.type === 'text') {
+        dom.setAttribute('data', Str.htmlEncode(dom.data));
+    }
```

Comment:
Ah I think this actually could use a code comment to explain why we are html encoding. It's not really clear.

Reply:
done